_maps/cyberiad.dm

```diff
@@ -18,9 +18,9 @@ z7 = empty
         #include "map_files\cyberiad\z2.dmm"
         #include "map_files\cyberiad\z3.dmm"
         #include "map_files\cyberiad\z4.dmm"
-        #include "map_files\cyberiad\z5.dmm"
-        #include "map_files\cyberiad\z6.dmm"
```

Comment:
This one isn't ready to be ripped out yet; it has the Russian direlict, Ruski DJ, and syndicate outpost.